tgui/packages/tgui/interfaces/RiotGear.js

```diff
@@ -0,0 +1,211 @@
+            </Box>
+          </Modal>
+        )}
+        {(!maxSecAccess && !cooldown && !(!authed)) && (
+          <Modal
+            fontSize="20px"
+            mr={2}
+            p={3}>
+            <Box>
+              Armoury Access Has Been Authorised!
+            </Box>
+          </Modal>
+        )}
+        {nameModal && (
+          <Modal
+            fontSize="20px"
+            mr={2}
+            p={3}>
+            <Box>
+              Authorisation Already Issued By User!
+            </Box>
+          </Modal>
+        )}
+        {printModal && (
+          <Modal
+            fontSize="20px"
+            mr={2}
+            p={3}>
+            <Box>
+              User Fingerprint ID Already On File!
+            </Box>
+          </Modal>
+        )}
+
+        <Section>
+          <Flex>
+            <Flex.Item width="50%">
+              <LabeledControls.Item>
+                <Button
+                  width={30}
+                  height={3.5}
+                  textAlign={"center"}
+                  fontSize={2}
+                  color={existingAuthorisation ? 'blue' : 'green'}
+                  disabled={authed ? true : false}
+                  onClick={() => (maxSecAccess
+                    ? act('HoS-Authorise')
+                    : ((nameOnFile === printOnFile)
+                      ? (existingAuthorisation
+                        ? act('Repeal')
+                        : act('Authorise'))
+                      : (nameOnFile
+                        ? displayNameModal(true)
+                        : displayPrintModal(true)))
+                  )}>
+                  <Flex width={30} height={3.5} align="center" justify="center">
+                    <Flex.Item>
+                      {existingAuthorisation ? 'Repeal' : 'Authorise'}
+                    </Flex.Item>
+                  </Flex>
+                </Button>
+              </LabeledControls.Item>
+            </Flex.Item>
+            <Flex.Item width="50%">
+              <LabeledControls.Item>
+                <Button
+                  width={30}
+                  height={3.5}
+                  textAlign={"center"}
+                  fontSize={2}
+                  color={'red'}
+                  disabled={authed ? false : true}
+                  onClick={() => (maxSecAccess && act('Deauthorise'))}>
+                  <Flex width={30} height={3.5} align="center" justify="center">
+                    <Flex.Item>
+                      Deauthorise
+                    </Flex.Item>
+                  </Flex>
+                </Button>
+              </LabeledControls.Item>
+            </Flex.Item>
+          </Flex>
+        </Section>
+        <Section
+          mb={0}
+          title="Authorisations:"
+          style={{ 'border-bottom': '2px solid rgba(51, 51, 51, 0.4);' }}
+        >
+          <Flex className="cloning-console__flex__head">
```

Comment:
I'm not really sure what this does, but this isn't the cloning console so it probably shouldn't be here

Reply:
`className="cloning-console__flex__head"` makes reference to the styles used by the cloning console's UI, used for the lists. I felt it better to reuse that instead of copying all of the style data from that file to here.